setup.py

```diff
@@ -33,32 +33,37 @@ def read(path):
     name='sphinx-csv-filter',
     version='0.3.0',
     url='https://github.com/crate/sphinx_csv_filter',
-    author='CRATE Technology GmbH',
+    author='Crate.IO GmbH',
     author_email='office@crate.io',
     package_dir={'': 'src'},
-    description='Sphinx CSV filter extension for Crate Documentation',
+    description='A CSV filter directive for docutils and Sphinx, that extends the '
+                '"csv-table" reStructuredText directive to add row filtering options.',
     long_description=read('README.rst'),
     long_description_content_type='text/x-rst',
     platforms=['any'],
     license='Apache License 2.0',
-    keywords='crate sphinx csv',
+    keywords='sphinx csv filter docutils directive plugin '
+             'csv-table csv-filter restructuredtext',
     packages=find_packages('src'),
     namespace_packages=['crate'],
     entry_points={},
     extras_require={
         "development": [
+            "build",
             "setuptools",
-            "wheel",
             "twine",
+            "wheel",
         ],
         "testing": [
+            "flake8<5",
             "pytest",
-            "flake8",
+            "pytest-cov",
             "pytest-flake8",
             "pytest-isort",
         ]
     },
     install_requires=[
+        "docutils<0.19",
         "Sphinx",
     ],
     python_requires=">=3.3",
```

Comment:
Make it `>=3.6`, as older versions aren't set up in the test matrix?

Reply:
Good catch. Nevertheless, this package has a number of dependents, and I wanted to run a maintenance release only, without adjusting this metadata attribute. I will be happy to bring in a corresponding update afterwards.

[^1]: https://github.com/crate/sphinx_csv_filter/network/dependents